### ISSUE TITLE
🐞 SPFx v1.17 - PropertyPaneCheckbox missing ariaLabel for accessibility

### ISSUE THREAD

**Issue by andrewconnell** [SharePoint/sp-dev-docs#8876]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

None

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- Chrome v111.0.5563.146
- SPFx v1.17
- Node.js v16.18.0

### Describe the bug / error

The SPFx v1.17 release added the `ariaLabel` property to the `PropertyPaneLink` for accessibility reasons.

Looking at the other controls, it appears to be missing on the `PropertyPaneCheckbox` control… shouldn’t it be on there too?

### Steps to reproduce

- create web part project
- add `PropertyPaneCheckbox` control to property pane
- observe the `ariaLabel` property isn't present

### Expected behavior

- same `ariaLabel` property present on `PropertyPaneCheckbox` control just like other properties

**Comment by andrewconnell** [comment]
Looks like this was fixed in SPFx v1.19...

https://learn.microsoft.com/en-us/sharepoint/dev/spfx/release-1.19#arialabel-property-for-the-property-panes-checkbox-control
